src/hotspot/share/gc/g1/g1SegmentedArrayFreeMemoryTask.hpp

```diff
@@ -82,14 +83,14 @@ class G1CardSetFreeMemoryTask : public G1ServiceTask {
   jlong reschedule_delay_ms() const;
 
 public:
-  explicit G1CardSetFreeMemoryTask(const char* name);
+  explicit G1SegmentedArrayFreeMemoryTask(const char* name);
```

Comment:
Yes, it will be used by some other places, e.g. evecuation failure processing. This class and related others will be refactored further to be more generic after the move and rename.